# Message pour les données de capteurs consolidées
Header header
sensor_msgs/LaserScan lidar_scan
geometry_msgs/Twist odometry_velocity
float64[] encoder_values
float64[] additional_sensor_data
string sensor_status